### PR TITLE
Cn 0435 scripts

### PR DESCRIPTION
**Pull request by msuciu**
Add python scripts for CN-0435 project.

- Add basic python scripts to read or write MODBUS registers from CN-0414 and/or CN-0418 boards.
- Add main scripts to interact via MODBUS commands with one or more CN-0414 or CN-0418 boards which compose a DCS or a PLC system.

### CODE REVIEW

File: projects/ADuCM3029_demo_cn0435/Scripts/Basic MODBUS Examples/read_adc_analog_input_registers.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""Read analog input registers to determine ADC channels code."""
```
Comment (mirceacaprioru) [inline projects/ADuCM3029_demo_cn0435/Scripts/Basic MODBUS Examples/read_adc_analog_input_registers.py]: Please limit the commit message to 75 character/line.

File: projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py [hunk 2]
```diff
@@ -0,0 +1,634 @@
...
+__version__ = '0.1'
+__author__ = 'Mihai Ionut Suciu'
+__status__ = 'Development'
+
+from time import sleep
+from colorama import init, Fore
+import dcs_cn0435_utilities as utilities
+
+init(autoreset=True)
+
```
Comment (mirceacaprioru) [inline projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py]: nitpick: extra line

File: projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py [hunk 3]
```diff
@@ -0,0 +1,634 @@
...
+            Fore.YELLOW + 'HART IN 21', Fore.YELLOW + 'HART IN 22',
+            Fore.YELLOW + 'HART IN 23', Fore.YELLOW + 'HART IN 24',
+            Fore.YELLOW + 'HART IN 25', Fore.YELLOW + 'HART IN 26',
+            Fore.YELLOW + 'HART IN 27', Fore.YELLOW + 'HART IN 28',
+            Fore.YELLOW + 'HART IN 29', Fore.YELLOW + 'HART IN 30']
+        utilities.print_table(
+            register_address, registers_number, labels, registers)
+    return registers
+
+
```
Comment (mirceacaprioru) [inline projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py]: nitpick: extra line

File: projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py [hunk 4]
```diff
@@ -0,0 +1,634 @@
...
+                currents = read_current_channels(global_data, cs_address)
+                print(Fore.GREEN + " ".join(
+                    str('{:+.5f} A'.format(element)) for element in currents))
+            except KeyboardInterrupt:
+                break
+            except OSError as error_message:
+                print(Fore.RED + 'Timeout error!', error_message)
+            except ValueError as error_message:
+                print(Fore.RED + 'Register operation error!', error_message)
+
```
Comment (mirceacaprioru) [inline projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py]: same as above

File: projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py (hunk 2 above)
Comment (msuciu) [reply projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py]: Two lines are necessary to respect python linting.

File: projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py (hunk 3 above)
Comment (msuciu) [reply projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py]: Two lines are necessary to respect python linting.

File: projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py (hunk 4 above)
Comment (msuciu) [reply projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py]: Two lines are necessary to respect python linting.

**Comment by msuciu** [comment]
V0 -> V1:
- Updated commit messages to respect the length of 75 characters.

File: projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py (hunk 4 above)
Comment (mirceacaprioru) [reply projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py]: ack, not familiar with python code.

**Comment by mirceacaprioru** [comment]
will merge later today if there are no objections.

**Pull request merged**